code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm

```diff
@@ -18,16 +18,24 @@ GLOBAL_LIST_EMPTY(solargrubs)
 	value = CATALOGUER_REWARD_EASY
 
 #define SINK_POWER 1
+var/global/list/moth_amount = 0 // Chompstation Addition, Rykka waz here. *pawstamp*; CitRP: It's for the moth spawn thingy
```

Comment:
GLOBAL_VAR_INIT(moth_amount, 0)

Reply:
Ok, I don't know what you wanted me to do, but it just broke shit.